Godeps/_workspace/src/k8s.io/kubernetes/pkg/runtime/deep_copy_generator.go

```diff
@@ -0,0 +1,609 @@
+/*
```

Comment:
Is the follow up issue list linked?

On Wed, Apr 20, 2016 at 10:22 PM, David Eads <notifications@github.com>
wrote:

> In
> Godeps/_workspace/src/k8s.io/kubernetes/pkg/runtime/deep_copy_generator.go
> <https://github.com/openshift/origin/pull/8484#discussion_r60518325>:
>
> > @@ -0,0 +1,609 @@
> > +/*
>
> Why?
>
> At the time? Time. The new deep copy generator had multiple bad
> assumptions about package availability, the old deep copy generator worked,
> and 1.2 was going to be done two days after I finished this so I wanted
> reviews, merge, then fix-up.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8484/files/c1afa8b0bd73fd9704f63c937bb351675987345a..7b2c760d241e2c46d87e4ce977ea65f06644e999#r60518325>
>